app/views/layouts/application.slim

```diff
@@ -7,14 +7,17 @@ html
     = stylesheet_link_tag 'application', media: 'all'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   body
+    p
+      = link_to 'new user by devise', new_user_registration_path
```

Comment:
but it was the same route, that's why I asked. For user creation using UsersController it is another route